pkg/synopsysctl/cmd_create.go

```diff
@@ -526,11 +535,165 @@ var createOpsSightNativeCmd = &cobra.Command{
 	},
 }
 
+// createCmd creates a Polaris instance
+var createPolarisCmd = &cobra.Command{
+	Use:           "polaris",
+	Example:       "synopsysctl create polaris <name> -n <namespace>",
+	Short:         "Create a Polaris instance",
+	SilenceUsage:  true,
+	SilenceErrors: true,
+	Args: func(cmd *cobra.Command, args []string) error {
+		// Check the Number of Arguments
+		if len(args) != 0 {
+			cmd.Help()
+			return fmt.Errorf("this command takes 0 argument")
+		}
+		return nil
+	},
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		err := createPolarisCobraHelper.SetPredefinedCRSpec("")
+		if err != nil {
+			cmd.Help()
+			return err
+		}
+		cobra.MarkFlagRequired(cmd.Flags(), "version")
+		cobra.MarkFlagRequired(cmd.Flags(), "environment-dns")
+		cobra.MarkFlagRequired(cmd.Flags(), "environment-name")
+		cobra.MarkFlagRequired(cmd.Flags(), "postgres-username")
+		cobra.MarkFlagRequired(cmd.Flags(), "postgres-password")
+
+		cobra.MarkFlagRequired(cmd.Flags(), "smtp-host")
+		cobra.MarkFlagRequired(cmd.Flags(), "smtp-port")
+		cobra.MarkFlagRequired(cmd.Flags(), "smtp-username")
+		cobra.MarkFlagRequired(cmd.Flags(), "smtp-password")
+
+		return nil
+	},
+	RunE: func(cmd *cobra.Command, args []string) error {
+		polarisObj, err := updatePolarisSpecWithFlags(cmd, namespace)
+		if err != nil {
+			return err
+		}
+
+		myscheme := runtime.NewScheme()
+		_ = clientgoscheme.AddToScheme(myscheme)
+
+		// create controller runtime client
+		cl, _ := runtimeclient.New(restconfig, runtimeclient.Options{
+			Scheme: myscheme,
+		})
+
+		// Marshal Polaris
+		polarisByte, err := json.Marshal(polarisObj)
+		if err != nil {
+			return err
+		}
+
+		polarisSecret := &v1.Secret{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "polaris",
+				Namespace: namespace,
+			},
+			Data: map[string][]byte{
+				"polaris": polarisByte,
+			},
+		}
+
+		// Create secret
+		polarisSecret, err = kubeClient.CoreV1().Secrets(namespace).Create(polarisSecret)
+		if err != nil {
+			return err
+		}
+
+		// Start reconcile process
+		if _, err := flying_dutchman.MetaReconcile(polarisSecret, &polaris.PolarisReconciler{
+			Client:      cl,
+			Scheme:      myscheme,
+			IsDryRun:    false,
+			IsOpenShift: false,
+			BaseUrl:     baseUrl,
+		}); err != nil {
+			return err
+		}
+
+		return nil
+	},
+}
+
+//createPolarisNativeCmd prints the Kubernetes resources for creating a Polaris instance
+var createPolarisNativeCmd = &cobra.Command{
+	Use:           "native NAME",
+	Example:       "synopsysctl create polaris native <name>",
+	Short:         "Print the Kubernetes resources for creating a Polaris instance",
+	SilenceUsage:  true,
+	SilenceErrors: true,
+	Args: func(cmd *cobra.Command, args []string) error {
+		// Check the Number of Arguments
+		if len(args) != 0 {
```

Comment:
The "Use" parameter above implies this command can take in one argument.

Reply:
fixed